packages/server/app/Services/Ws.ts

```diff
@@ -1,15 +1,12 @@
 import Env from '@ioc:Adonis/Core/Env';
 import AdonisServer from '@ioc:Adonis/Core/Server';
-import {
-    ChatClientToServerEvents,
-    ChatServerToClientEvents,
-} from '@musicroom/types';
-import { createAdapter } from '@socket.io/redis-adapter';
+import { AllEvents } from '@musicroom/types/src/websockets';
+import { createAdapter, RedisAdapter } from '@socket.io/redis-adapter';
 import { RedisClient } from 'redis';
 import { Server } from 'socket.io';
 
 class Ws {
-    public io: Server<ChatClientToServerEvents, ChatServerToClientEvents>;
+    public io: Server<AllEvents>;
```

Comment:
We should separate messages sent by the client to the server and the messages sent by the server to the client.
That is, use two separate types and provide them as two different generics.